### PR TITLE
Update guide for using modelevaluation.org

### PR DESCRIPTION
**Pull request by SeanBryan51**

### CODE REVIEW

**Review by ccarouge — COMMENTED** [review]
@SeanBryan51 I have added a commit with quite a few changes. Please have a look and update as you like.

File: docs/user_guide/index.md [hunk 1]
```diff
@@ -135,15 +135,67 @@ The output files and log files for all tasks are stored in the `runs/site/output
...
+        - Optionally, in **the comments**, you may also want to include the URL to the Github repository containing the benchcab configuration file used to run `benchcab` and any other information needed to reproduce the outputs.
+
+    4. Under "Model Output Files", click **Upload Files**. This should prompt you to select the model outputs you want to upload from your file system. We recommend users to make their model outputs public to download by checking **Downloadable by other users**.
+        <figure markdown>
+          ![Public output](../assets/model_evaluation/Public%20output.png){ width="300" }
+          <figcaption>Make model output public</figcaption>
+        </figure>
+
+    5. Under "Benchmarks", you may need to add a benchmark depending on the experiment chosen. This is an error and will be fixed soon.
+        - **Five site test** and **Forty two site test**: a benchmark is required to run the analysis for the `Five site test` experiment. You can use [this model profile][benchmark_eg] as a benchmark for this experiment.
```
Comment (SeanBryan51) [inline docs/user_guide/index.md:183]: @ccarouge a link is missing here for `benchmark_eg`

**Comment by ccarouge** [comment]
@SeanBryan51 I have fixed the links. Is this all good to you?

**Comment by SeanBryan51** [comment]
Yes all good. Thanks for adding the images, looks much better!

**Review by ccarouge — APPROVED** [review]

**Pull request merged**